demisto_sdk/tests/integration_tests/format_integration_test.py

```diff
@@ -226,3 +230,36 @@ def _verify_conf_json_modified(test_playbooks: List, yml_title: str, conf_json_p
                 )
     except Exception:
         raise
+
+
+DEMISTO_SDK_PATH = join(git_path(), "demisto_sdk")
+
+
+def test_integration_format_remove_playbook_sourceplaybookid():
+    """
+    Given
+    - Playbook with field  `sourceplaybookid`.
+    - destination_path to write the formatted playbook to.
+
+    When
+    - Running the format command.
+
+    Then
+    - Ensure 'sourceplaybookid' was deleted from the yml file.
+    """
+    source_playbook_path = join(DEMISTO_SDK_PATH, "tests/test_files/format_new_playbook_copy.yml")
+    output_playbook_path = join(DEMISTO_SDK_PATH, "tests/test_files/new_format_new_playbook_copy.yml")
+
+    runner = CliRunner(mix_stderr=False)
+    arguments = [
+        'format',
+        '-i', source_playbook_path,
+        '-o', output_playbook_path
+    ]
+    runner.invoke(main, arguments)
+
```

Comment:
assert exit_code == 0
assert no exception
assert no stderr
assert that expected output (Starting updates... Finished updates..., etc..) is in stdout